packages/ui-components/src/components/UIButton/UIIconButton.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import type { IButtonProps as IBaseButtonProps, IButtonStyles } from '@fluentui/react';
+import { IconButton } from '@fluentui/react';
+
+import { UIContextualMenu } from '../UIContextualMenu';
+
+export enum UIIconButtonSizes {
+    Default = 'Default',
+    Wide = 'Wide'
+}
+
+export interface ButtonProps extends IBaseButtonProps {
+    sizeType?: UIIconButtonSizes;
+}
+
+/**
+ * UIIconButton component.
+ * based on https://developer.microsoft.com/en-us/fluentui#/controls/web/button
+ *
+ * @exports
+ * @class UIIconButton
+ * @extends {React.Component<ButtonProps, {}>}
+ */
+export class UIIconButton extends React.Component<ButtonProps, {}> {
+    /**
+     * Initializes component properties.
+     * @param {ButtonProps} props
+     */
+    public constructor(props: ButtonProps) {
+        super(props);
+    }
+
+    /**
+     * Method which returns button interaction background with including fallback for olde VSCode or BAS versions.
```

Comment:
Is this still required? please check which exact versions require the fallback and check if we still support them

Reply:
as I checked, looks like we still need it. `--vscode-toolbar-hoverBackground` was presenter at VSCode version about ~`1.56`.
But as I checked variable `--vscode-toolbar-hoverBackground` still not presenter in BAS environment:
![image](https://user-images.githubusercontent.com/90789422/188813266-90a88213-dd58-4d52-a2e3-25748455ea37.png)
In result fallback is required for BAS

Reply:
Thanks